src/qt_frontend/qt_frontend.cpp

```diff
@@ -1,12 +1,126 @@
 #include <QApplication>
+#include <QCommandLineParser>
 #include <settings.h>
+#include <cflags.h>
```

Comment:
I know that QCommandLineParser exists, but AFAIK it isn't as straightforward to have a repeating flag (like your `-vvvv`) as it is with cflags.